package.json

```diff
@@ -131,9 +131,9 @@
     ]
   },
   "dependencies": {
-    "@csstools/css-parser-algorithms": "^2.3.0",
+    "@csstools/css-parser-algorithms": "^2.3.1",
     "@csstools/css-tokenizer": "^2.1.1",
```

Comment:
It's weird that dependabot didn't increase this version.
The lock file is correct, but I also expected `^2.2.0` here

I don't have any immediate idea what might cause this.
Maybe this is because of config here, or a bug in the new grouping feature.

Reply:
Surely, we expect also to update to `"@csstools/css-tokenizer": "^2.2.0"`. I'll manually push a change.

By the way, thanks to `peerDependencies`, this will not be an actual harm, right?

```console
$ npm v @csstools/css-parser-algorithms peerDependencies
{ '@csstools/css-tokenizer': '^2.2.0' }
```

Reply:
Pushed 5527b6ff5

Reply:
Yes, I don't think it makes a practical difference, but it is unexpected.

Reply:
The thing that does bug me about this is that the real change was hidden.

It was the tokenizer that had new feature additions and the other two packages where releases after and because of the tokenizer minor release.

And yet in this PR we only see the two releases that aren't really interesting.
Composition like this makes it harder to review dependency updates.

Maybe this is a dependabot bug?

Reply:
I filed a bug report here : https://github.com/dependabot/dependabot-core/issues/7646